docs/source/incompressible_viscous_basics.rst

```diff
@@ -0,0 +1,80 @@
+Incompressible hydrodynamics solver
```

Comment:
I think you still need to include it in `index.rst`.

Reply:
Done